src/CFamily/Rules/ICFamilyRulesConfig.cs

```diff
@@ -50,6 +51,15 @@ public class RuleMetadata
 
         [JsonProperty("compatibleLanguages")]
         public string[] CompatibleLanguages { get; set; }
+
+        [JsonProperty("code")]
+        public Code Code { get; set; }
```

Comment:
The name `Code` looks odd. Maybe rename the class, but keep the json property name (since we can't change that)?

Reply:
It looks odd but I prefer to have the same name it makes maintenance easier.